doc/intro.md

```diff
@@ -0,0 +1,3 @@
+# Introduction to tic-tac-toe
```

Comment:
If you have a good read me you might not need this. Also there are still those files that Tony ask you to remove since they are in your gitignore file